range_max: 10
range_min: 0.1
angle_max: -3
angle_min: 3
